frontend/src/pushapi.spec.js

```diff
@@ -366,6 +366,21 @@ describe('PushApi', function () {
       expect(connectMethod).toHaveBeenCalledWith(subscriber);
     });
 
+    it('not double keep-alive interval after channel expire', function () {
```

Comment:
Maybe something like: it('uses single keep-alive for all channels'...   sounds better.